env.template

```diff
@@ -89,12 +88,22 @@ SSL_CA_PATH=scos_test_ca.crt
 SSL_CERT_PATH=sensor01.pem
 SSL_KEY_PATH=sensor01.pem
 
+# Device made available to SCOS Plugins, e.g. for getting SMART data
+SSD_DEVICE="$(lsblk -I 8 -npdo KNAME)"
```

Comment:
This command isn't returning anything on the sensors or my laptop

Reply:
Replaced with a plaintext default value which can be changed when configuring a sensor. 

Note also that I made changes in scos-actions (NTIA/scos-actions#126) which will result in  a default value of `/dev/nvme0n1` being used by the SEA action if the environment variable is not provided.